GiniSwitchSDK/Example/GiniTariffSDK/ExtractionsViewController.swift

```diff
@@ -62,7 +62,15 @@ extension ExtractionsViewController: UITableViewDataSource {
         cell.nameLabel.text = extraction?.name ?? ""
         cell.valueTextField.text = extraction?.valueString ?? ""
         cell.onExtractionChange = { (newValue: String) -> Void in
-            extraction?.value = ExtractionValue(value: newValue as AnyObject, unit: extraction?.value.unit)
+            // Since the data is shown in a text field, all entries are strings. However, it is
+            // important to NOT change the data type in the extraction collection
+            // (otherwise the backend might reject the data)
```

Comment:
Just a comment. In case that you send "218" instead of 218, the backend accept it, right? I imagine that you just add this as a double check.

Reply:
Not really. The backend is very strict about data types. That's the whole point of the PR.

Reply:
No. The backend is quite strict about data types. That's the whole point of this PR.